README.md

```diff
@@ -219,6 +219,8 @@ Please see the netdev_stdlib docs https://github.com/puppetlabs/netdev_stdlib/bl
 * [`ios_aaa_new_model`](#ios_aaa_new_model): Enable aaa new model on device
 * [`ios_aaa_session_id`](#ios_aaa_session_id): Configure aaa session id on device
 * [`ios_config`](#ios_config): Execute an arbitrary configuration against the cisco_ios device with or without a check for idempotency.
+* [`ios_ntp_access_group`](#ios_ntp_access_group): Configures NTP Access Group for network devices.
+* [`ios_ntp_config`](#ios_ntp_config): Extends the ntp type
```

Comment:
These new types will need added to the test matrix, might be best creating a ticket as there is currently a PR up to move the test matrix to it's own file - if it gets merged first then rebasing and including the updates as part of this PR will be best

Reply:
ios_ntp_config still needs added to the test matrix